Source/MipsExecutor.cpp

```diff
@@ -148,10 +148,8 @@ bool CMipsExecutor::MustBreak() const
 {
 	uint32 currentPc = m_context.m_pAddrTranslator(&m_context, m_context.m_State.nPC);
 	CBasicBlock* block = FindBlockAt(currentPc);
-	for(auto breakPointIterator(m_context.m_breakpoints.begin());
-		breakPointIterator != m_context.m_breakpoints.end(); breakPointIterator++)
+	for(auto breakPointAddress : m_context)

```

Comment:
you're right, i missed that it was inside a debug if preprocessor, so i assumed it compiled when I tested it, when in fact it wasn't even being included.